pkg/manager/task_import.go

```diff
@@ -42,15 +42,36 @@ type ImportTask struct {
 	fileNamingAlgorithm models.HashAlgorithm
 }
 
-func CreateImportTask(a models.HashAlgorithm, input models.ImportObjectsInput) *ImportTask {
+func CreateImportTask(a models.HashAlgorithm, input models.ImportObjectsInput) (*ImportTask, error) {
+	baseDir, err := instance.Paths.Generated.TempDir("import")
+	if err != nil {
+		logger.Errorf("error creating temporary directory for import: %s", err.Error())
+		return nil, err
+	}
+
+	tmpZip := ""
+	if input.File.File != nil {
+		tmpZip := filepath.Join(baseDir, "import.zip")
```

Comment:
sorry made a typo here when pasting
```suggestion
		tmpZip = filepath.Join(baseDir, "import.zip")
```